src/test/compile-fail/issue-28324.rs

```diff
@@ -15,6 +15,6 @@ extern {
 }
 
 pub static BAZ: u32 = *&error_message_count;
-//~^ ERROR cannot refer to other statics by value
+//~^ ERROR constant evaluation error
```

Comment:
Can you add another line with `//~| read from foreign (extern) static` just to make it clear what error it triggers?

Reply:
@eddyb That gives a "not found" error unfortunately.

Reply:
...since I don't think it's actually part of the error message.

Reply:
Wait, what? What do you get if you compile that test locally, directly?